client/src/views/Help.vue

```diff
@@ -28,20 +28,20 @@
-                    <br><Card :number="1" :color="'red'"></Card><br><br><br>
+                    <br><Card :number="'D2'" :color="'red'"></Card><br><br><br>
                     <strong> Draw Two </strong> – When a person places this card, the next player will have to pick up two cards and forfeit his/her turn. It can only be played on a card that matches by color, or on another Draw Two. If turned up at the beginning of play, the first player draws two cards and gets skipped.<br><br><br><br><br>
-                    <br><Card :number="1" :color="'red'"></Card><br><br>
+                    <br><Card :number="'W'" :color="'red'"></Card><br><br>
                     <strong> Wild </strong> – This card represents all four colors, and can be placed on any card. The player has to state which color it will represent for the next player. It can be played regardless of whether another card is available. If turned up at the beginning of play, the first player chooses what color to continue play.<br><br><br><br><br>
-                    <br><Card :number="1" :color="'red'"></Card><br><br>
+                    <br><Card :number="'W4'" :color="'red'"></Card><br><br>
                     <strong> Wild Draw Four </strong> – This acts just like the wild card except that the next player also has to draw four cards as well as forfeit his/her turn. With this card, you must have no other alternative cards to play that matches the color of the card previously played. If you play this card illegally, you may be challenged by the other player to show your hand to him/her. If guilty, you need to draw 4 cards. If not, the challenger needs to draw 6 cards instead. If turned up at the beginning of play, return this card to the Draw pile, shuffle, and turn up a new one.<br><br><br><br><br>
-                    <br><Card :number="1" :color="'red'"></Card><br>
+                    <!-- <br><Card :number="1" :color="'red'"></Card><br>
                     <strong> Wild Swap Hands Card </strong> – This is a powerful card that enables you to swap the cards in your hand with anyone of your choice. Being a Wild card, it has the same attributes, namely, you can choose to either play it or not, at any of your turns, and also choose the color of play. Ideally, you would probably want to swap your cards with the player who has the least number of cards! Being a powerful Action card, only one is provided per deck. If turned up at the beginning of the game, the first player to start gets to choose the color and also swap his hands with another person. Note: Being a Wild card, the same rule applies to the Swap Hands card. Once you play this card and swapped your hands with another player, your turn is up, and you are not allowed to put down any card immediately after that until your turn arrives again.<br><br><br><br><br>
```

Comment:
do we need to keep this text if you are commenting it out?

Reply:
Are we going to use those types of cards? if not then yeah we can delete it